content/guides/repl/data_visualization_at_the_repl.adoc

```diff
@@ -63,7 +63,7 @@ user=> (mapv number-summary [5 6 7 12 28 42])
 ----
 
 We can use the `https://clojure.github.io/clojure/clojure.pprint-api.html[clojure.pprint]` lib
- to print the result in a more 'visual' format:
```

Comment:
There were 2 instances of 'quoted' terms and 2 instances of "quoted" terms in this doc, so I harmonized them, using double quotes as those are more common in English prose.